README.md

```diff
@@ -98,4 +98,35 @@ $ ansible-playbook my-ansible-playbook.yml -i ansible-inventory.ini
 
 ## Module 4 stretch goal
 
-$ poetry add gunicorn
\ No newline at end of file
+$ poetry add gunicorn
+
+## Run in Docker
+
+#### Development Mode
+
+1. Build the development image
+   '''bash
+   docker build --target development --tag todo-app:dev .
+   '''
```

Comment:
Minor but this doesn't seem to format sensibly (at least on GitHub/in VSCode) - I think you want:
````suggestion
   ```bash
   docker build --target development --tag todo-app:dev .
   ```
````